contracts/libs/Types.sol

```diff
@@ -18,7 +18,7 @@ library Types {
         bytes32[][] stateWitnesses;
         uint256[4][] pubkeysSender;
         bytes32[][] pubkeyWitnessesSender;
-        uint256[4][] pubkeysReceiver;
+        bytes32[] pubkeysReceiver;
```

Comment:
Nitpick: What do you think we rename pubkeysReceiver to pubkeyHashesReceiver to emphasize the hash part? Since we have `uint256[4][] pubkeysSender` and `bytes32[] pubkeysReceiver` together here, they are all named pubkeysX but mean differently.

Reply:
Makes sense to me, will rename